bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+import numpy as np
+import h5py
+import operator
+import argparse
+import logging
+from awkde import GaussianKDE
+from pycbc import init_logging
+import pycbc.conversions as convert
+from sklearn.model_selection import KFold
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file')
+parser.add_argument('--template-file')
```

Comment:
Is the option template-file required?  Ie can the code run without a template file? If not, the option should be required